test/extensions/filters/udp/udp_proxy/udp_proxy_integration_test.cc

```diff
@@ -103,8 +133,21 @@ TEST_P(UdpProxyIntegrationTest, HelloWorldOnLoopback) {
   requestResponseWithListenerAddress(*listener_address);
 }
 
-// Verifies calling sendmsg with a non-local address. Note that this test is only fully complete for
-// IPv4. See the comment below for more details.
+// Test with large packet sizes.
+TEST_P(UdpProxyIntegrationTest, LargePacketSizesOnLoopback) {
+  // The following tests large packets end to end. We use a size of 32K as this is larger than
+  // the default GRO receive buffer size of 1500 x 16 = 24000 bytes.
+  const uint64_t max_rx_datagram_size = 32768;
```

Comment:
should we make num_packets_per_mmsg_call public so we can check it here, and other places we assume 16?